tests/test_onco_print.py

```diff
@@ -94,3 +120,38 @@ def assert_callback(prop_value, nclicks, input_value):
         component_base=COMPONENT_REACT_BASE,
         data=TEST_DATA
     )
+
+    driver = dash_threaded.driver
+    # assert there is a legend (bar)
+    legend = driver.find_elements_by_class_name('legendbar')
+    assert len(legend) != 0
+
+    # trigger change of the component prop
+    btn = wait_for_element_by_css_selector(driver, '#test-{}-btn'.format(APP_NAME))
+    btn.click()
+
+    # assert there is no more legend (bar)
+    legend = driver.find_elements_by_class_name('legendbar')
```

Comment:
```suggestion
    legend = wait_for_elements_by_css_selector(driver, '.legendbar')
```

Reply:
Reading function definition for [`wait_for_elements_by_css_selector()`](https://github.com/T4rk1n/pytest-dash/blob/2ea221fad6913f168693212986b28b576fca091c/pytest_dash/wait_for.py#L44). I wish it was called something *more* different, such as "wait_for_all_elements" but I understand the function name is already very long ;) @T4rk1n

Reply:
Oh, I think I just inferred the meaning of "Add suggestion to batch"! We can probably include several suggestions into one commit, which I should have done here, so it makes sense conceptually *and* CI tests pass!
/cc @shammamah